optaplanner-examples/src/main/resources/org/optaplanner/examples/app/benchmark/ruinWithConstantTimeOptaPlannerBenchmarkConfigTemplate.xml.ftl

```diff
@@ -5,9 +5,9 @@
 	<inheritedSolverBenchmark>
 		<solver>
 			<termination>
-				<minutesSpentLimit>15</minutesSpentLimit>
+				<minutesSpentLimit>10</minutesSpentLimit>
 			</termination>
-			<subSingleCount>10</subSingleCount>
+			<subSingleCount>5</subSingleCount>
```

Comment:
I respect that you want to be thorough, but considering how many benchmarks you need to run, are you sure you want to multiply that number by 5?

Reply:
Don't get me wrong - 5 is good, for results that are more statistically relevant. :-) But unless you have a very powerful machine, you will be waiting for a long time.